doc/classes/RenderingDevice.xml

```diff
@@ -774,6 +774,14 @@
 				Returns [code]true[/code] if the [param texture] is valid, [code]false[/code] otherwise.
 			</description>
 		</method>
+		<method name="texture_native_handle">
+			<return type="int" />
+			<param index="0" name="texture" type="RID" />
+			<description>
+				Returns the internal graphics handle for this texture object. For use when communicating with 3rd party APIs mostly with GDExternal.
+				[b]Note:[/b] This functions returns a [code]uint64_t[/code] which internally maps to a [code]GLuint[/code] (OpenGL) or [code]VkImage[/code] (Vulkan).
```

Comment:
```suggestion
				[b]Note:[/b] This function returns a [code]uint64_t[/code] which internally maps to a [code]GLuint[/code] (OpenGL) or [code]VkImage[/code] (Vulkan).
```

Reply:
You missed that one.

Reply:
wtf, it's changed locally, let me see whats going on...

Reply:
Ah I must really have been sleeping when I fixed this (or rushing most likely), I removed the wrong s :)